dotnet/SimpleSauce.Test/BaselineSeleniumAcceptanceTests.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using Simple.Sauce;
+
+namespace SimpleSauce.Test
+{
+    [TestClass]
+    [TestCategory("Acceptance")]
+    public class BaselineSeleniumAcceptanceTests
+    {
+        private Dictionary<string, object> _sauceOptions;
+        private IWebDriver _driver;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_driver' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4739713)